js/lib/beautify-html.js

```diff
@@ -161,6 +163,7 @@
             'acronym', 'address', 'big', 'dt', 'ins', 'strike', 'tt',
             'pre',
         ];
+        content_unformatted = options.content_unformatted || [];
```

Comment:
Please move `'pre'` from `unformatted`, into the default for content_unformatted and add some tests for it based on #928.